packages/server-core/src/user/identity-provider/identity-provider.class.ts

```diff
@@ -38,7 +41,7 @@ export class IdentityProvider extends Service {
    * @param params
    * @returns accessToken
    */
-  async create(data: any, params: Params): Promise<any> {
+  async create(data: any, params: Params = {}): Promise<T & { accessToken?: string }> {
```

Comment:
Like ``data:any`` can be changed

Reply:
yes, sorry i meant to assign you not request a review. I will provide some more details on what is needed for this PR soon.

Reply:
Great